WPMedia/fusion-news-theme-blocks
# TMEDIA-23 PoWa player script

## Description

Remove loading of PoWa player from the output type as it is included as part of the `embed_html` ANS field.


## Jira Ticket
- [TMEDA-23](https://arcpublishing.atlassian.net/browse/TMEDIA-23)

## Acceptance Criteria
1. Powa scripts are only loaded if the page contains a powa video player
2. If the page does not contain a powa video player, then the powa scripts should not load
3. From Gavin McDonald: We should not load powaDrive explicitly – instead, just let powaBoot.js do it. 

## Test Steps

To link the output type locally you **have** to use `FUSION_RELEASE=2.7.0`

1. Checkout this branch `git checkout TMEDIA-23-powa-player-script`
2. In fusion repo set your `FUSION_RELEASE` value in the `.env` file to be `2.7.0`
3. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/default-output-block,@wpmedia/video-player-block`
4. Visit the following pages and verify video's still play and PoWa player is loaded in the network tab
    * http://localhost/local/2020/01/14/tampa-reinvents-itself-to-show-off-its-shores/?_website=the-gazette
    * http://localhost/pf/demo-video-block-homepage/?_website=the-gazette
    * http://localhost/2019/09/17/global-kitchen-sink-article/?_website=the-gazette
    * http://localhost/video/2019/12/17/a-snowy-waterfall/?_website=the-gazette
5. Visit the following pages and verify PoWa player is not loaded as no video's on page
    * http://localhost/homepage?_website=the-gazette
    * http://localhost/2019/09/17/article-with-a-long-headline-lorem-ipsum-dolor-sit-amet-consectetur-adipiscing-elit/?_website=the-gazette

## Dependencies or Side Effects

* Requires `FUSION_RELEASE=2.7.0` to test locally

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
